File: webapp/airTicket/src/app/entity/ShipSpace.ts
// 舱位实体

import {Seat} from './Seat';

export class ShipSpace {

  id: number;

  described: string; // 描述

  primaried: boolean; // 是否为主舱位

  seats: Array<Seat>; // 座位集合

  constructor(described: string, primary: boolean) {
    this.described = described;
    this.primaried = primary;
  }
}

File: app/app/src/main/java/com/jdxiang/airTicket/entity/ShipSpace.java
package com.jdxiang.airTicket.entity;

import java.util.ArrayList;
import java.util.List;

public class ShipSpace {

    private Long id;

    private String described; // 描述

    private Boolean primaried; // 是否为主舱位

    private List<Seat> seats = new ArrayList<>(); // 座位集合

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getDescribed() {
        return described;
    }

    public void setDescribed(String described) {
        this.described = described;
    }

    public Boolean getPrimaried() {
        return primaried;
    }

    public void setPrimaried(Boolean primaried) {
        this.primaried = primaried;
    }

    public List<Seat> getSeats() {
        return seats;
    }

    public void setSeats(List<Seat> seats) {
        this.seats = seats;
    }
}
